src/main/java/org/prgrms/cream/domain/product/service/ProductService.java

```diff
@@ -54,7 +70,31 @@ private void modifyOption(Product product, String size) {
 
 	private Product findActiveProduct(Long id) {
```

Comment:
```suggestion
	public Product findActiveProduct(Long id) {
```

이 메서드가 나중에 다른 서비스 로직에서도 쓰일 것 같은 생각이 들었어요(User 에서도 똑같은 이슈(?)가 있었어요!)
그래서 public으로 만들어줘도 좋을 것 같다는 생각을 했습니다!
아니면 지금은 그대로 두고 나중에 그런 이슈가 발생한다면 수정하는 것도 괜찮을 것 같네요
부희님은 어떻게 생각하시나요??

Reply:
네! 구매와 판매 쪽에서 상품과 관계있느니 해당 상품이 존재하는지 확인하는 메서드를 사용하겠군요!! 다른 서비스와 공유하고 기본 기능이 아닌 확인하는 용도의 메서드니니 그런 부분도 구분지어줄 겸`protected` 접근 제어자를 사용하는 것이 어떨까요?